sandbox.sh

```diff
@@ -389,7 +389,7 @@ test_wasm() {
   ARG=$(ticket_handle)
   echo "param: \"$ARG\""
   sleep 5
-  deku-cli create-transaction ./data/0 wallet.json "$CONTRACT_ADDRESS" 100 "Pair \"$DUMMY_TICKET\" 0x" --arg="$ARG" --vm_flavor="Wasm" --tickets="Pair \"$DUMMY_TICKET\" 0x":100
+  deku-cli create-transaction ./data/0 wallet.json "$CONTRACT_ADDRESS" 100 "Pair \"$DUMMY_TICKET\" 0x" --arg="$ARG" --vm_flavor="Wasm" --tickets="Pair \"$DUMMY_TICKET\" 0x":100=0:0
```

Comment:
What the extra zero means?

Reply:
handle:index_inside_argument

Reply:
But what are the index for ?

Reply:
to have a pointer to ticket handle inside the argument bytes, in case we have a different ticket with the same handle already in the contract storage, so to avoid the conflicts we just rename the handles for now, what's why we need ticket -> handle mapping inside contract, and index + handle inside cli arguments. We need a proper argument and initial argument parsing functionality